media/js/firefox/australis/growth-firstrun-test2.js

```diff
@@ -0,0 +1,246 @@
+            }
+        });
+
+        // GA for both "what is an add-on?" links
+        $('.what-is-addon').on('click', function(e) {
+            var newTab = (this.target === '_blank' || e.metaKey || e.ctrlKey);
+            var href = this.href;
+            var parent = $(this).parents('div:first').attr('id');
+            var version = (parent === 'ui-final') ? 'final' : 'main';
+
+            if (newTab) {
+                window.gaTrack(['_trackEvent', '/firstrun/ Optimization f36', 'link click', 'what is an addon - ' + version]);
+            } else {
+                e.preventDefault();
+                window.gaTrack(['_trackEvent', '/firstrun/ Optimization f36', 'link click', 'what is an addon - ' + version], function() {
+                    window.location = href;
+                });
+            }
+        });
+
+        // Query if the UITour API is working before we start the tour
+        Mozilla.UITour.getConfiguration('sync', function() {
+            var suppressDoorhanger = Math.random() >= 0.5;
+
+            // GA
+            gaVariation = (suppressDoorhanger) ? 'a' : 'b';
+
+            window.gaTrack(['_setCustomVar', 7, 'first run tests', 'variation 3' + gaVariation, 2]);
+            window.gaTrack(['_trackEvent','/firstrun/ A/B Test', 'page load', 'variation 3' + gaVariation]);
+
+            tour = new Mozilla.BrowserTour({
+                id: $('#tour-page').data('telemetry'),
+                allowScroll: true,
+                suppressDoorhanger: suppressDoorhanger,
+                startTour: function() {
+                    // load iframe src's
+                    enableIframes();
+
+                    // load youtube
+                    initYouTube();
+
+                    // show contents and close button
+                    $('.mask-hidden').addClass('in');
+
+                    // mark tour as finished (but don't get rid of mask yet)
+                    tour.tourHasFinished = true;
+                },
+                cancelTour: function() {
+                    closeTour();
+                }
+            });
+
+            tour.init();
+
+            $('#close-tour').on('click', function() {
+                openFinalNotice(true);
+            });
+
+            // if doorhanger is not shown in tour, we need to display the
+            // contents & close button without user interaction
+            if (suppressDoorhanger) {
+                // load iframe src's
+                enableIframes();
+
+                // load youtube
+                initYouTube();
+
+                // show contents and close button
+                $('.mask-hidden').addClass('in-delayed');
+            }
+
+            // listen for clicks on "here" in alternate secondary heading
+            $('#highlight-addons').on('click', function(e) {
+                e.preventDefault();
+
+                Mozilla.UITour.showHighlight('addons', 'none');
```

Comment:
Same here [Bug 1049130](https://bugzilla.mozilla.org/show_bug.cgi?id=1049130) and probably any other places where it might be the first highlight shown.